modules/op-uninstalling-the-pipelines-operator.adoc

```diff
@@ -15,3 +15,8 @@ You can uninstall the {pipelines-title} Operator by using the *Administrator* pe
 . Click the *{pipelines-title}* Operator tile. The Operator tile indicates that the Operator is installed.
 
 . In the *{pipelines-title}* Operator description page, click *Uninstall*.
+
+[WARNING]
+====
+When you uninstall the Operator, all resources in the Pipeline namespace are lost including the secrets you configured with it.
```

Comment:
it's not "the Pipeline namespace". I believe it is the target namespace where {pipelines-shortname} is installed, by default, the `openshift-pipelines` namespace. We need to verify this with engineering. I have put a question into Slack.

Reply:
Alright, I see a thumbs up from you @vdemeester. How do you advise?

Reply:
@masauso-lungu I have confirmation from Pavol now - it is the namespace I described

Reply:
@mramendi Thank you for the update. I will address it accordingly